telegram/ext/_conversationhandler.py

```diff
@@ -675,12 +675,15 @@ def _schedule_job(
 
         try:
             # both job_queue & conversation_timeout are checked before calling _schedule_job
-            j_queue = application.job_queue
-            self.timeout_jobs[conversation_key] = j_queue.run_once(
-                self._trigger_timeout,
-                self.conversation_timeout,  # type: ignore[arg-type]
-                data=_ConversationTimeoutContext(conversation_key, update, application, context),
-            )
+            if application.job_queue:
+                j_queue = application.job_queue
+                self.timeout_jobs[conversation_key] = j_queue.run_once(
+                    self._trigger_timeout,
+                    self.conversation_timeout,  # type: ignore[arg-type]
+                    data=_ConversationTimeoutContext(
+                        conversation_key, update, application, context
+                    ),
+                )
```

Comment:
the comment in line 677 essentially means that we don't need an if-clause here :)

Reply:
Welp, the mypy-ptb pre commit run fails if I dont put if clause here... :thinking: